web/src/datavzrd.js

```diff
@@ -1351,6 +1350,14 @@ function decompress(data) {
 }
 
 export function sort(column, order, svg) {
+    const currentSortColumn = $('#table').bootstrapTable('getOptions').sortName;
+    const currentSortOrder = $('#table').bootstrapTable('getOptions').sortOrder;
+    if (currentSortColumn === column && currentSortOrder === order) {
```

Comment:
```suggestion
    const options = $('#table').bootstrapTable('getOptions');
    if (options.sortName === column && options.sortOrder === order) {
```

I think this looks a bit nicer.